src/sign-module.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Is required bash? or enough sh?

Reply:
Not sure, haven't tested it. It was specified in the provided document. I'll try to test it:
Not sure, haven't tested it. It was specified in the provided document: